CHANGELOG.md

```diff
@@ -3,6 +3,93 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.12.0
+
+_Aug 1, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+<!-- Blogpost for price changes, use it on tree view pro announcement -->
+
+[Upcoming changes to MUI X pricing in 2024](https://mui.com/blog/mui-x-sep-2024-price-update/)
```

Comment:
I assume we want a bigger explanation here regarding treeview-pro 😅 

Should we also mention charts-pro is coming soon?

Reply:
Not sure how to phrase this one
We should probably emphasize more the good news before linking to the blog post

```suggestion
- 💵 Our commercial offering is evolving!

  The pro plan is receiving two new packages:
  - `@mui/x-tree-view-pro` (available this week)
  - `@mui/x-charts-pro` (available in the coming weeks)
  
  As always, every feature released as part of the MIT plan will remain free and MIT licensed forever.
 
 This expansion of the pro plan comes with some adjustments to our pricing strategy. Learn more about those in the [Upcoming changes to MUI X pricing in 2024](https://mui.com/blog/mui-x-sep-2024-price-update/) blog post.
```